scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -969,7 +969,7 @@ class ScalaTestAntTask extends Task {
   // Class to hold data from <testsfile> elements.
   //
   private class TestsfileElement {
-    private var filename: String = null
+    private var filename: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.